src/Microsoft.Health.CosmosDb/Features/Storage/RetryExceptionPolicyFactory.cs

```diff
@@ -21,19 +22,20 @@ public RetryExceptionPolicyFactory(CosmosDataStoreConfiguration configuration)
             EnsureArg.IsNotNull(configuration, nameof(configuration));
 
             _maxNumberOfRetries = configuration.RetryOptions.MaxNumberOfRetries;
-            _minWaitTime = TimeSpan.FromSeconds(Math.Min(RetryStrategy.DefaultMinBackoff.TotalSeconds, configuration.RetryOptions.MaxWaitTimeInSeconds));
+            _minWaitTime = TimeSpan.FromSeconds(configuration.RetryOptions.MinWaitTimeInSeconds);
             _maxWaitTime = TimeSpan.FromSeconds(configuration.RetryOptions.MaxWaitTimeInSeconds);
```

Comment:
Looks like both `_minWaitTime` and `_maxWaitTime` are unused now. If so these vars can be removed